opba-protocols/sandboxes/hbci-sandbox/src/main/java/de/adorsys/opba/protocol/sandbox/hbci/protocol/authenticated/nonauthorized/AuthenticatedCustomMsg.java

```diff
@@ -80,7 +85,20 @@ private String handlePayment(HbciSandboxContext context) {
         if (RequestStatusUtil.isForPayment(context.getRequestData())) {
             context.setAccountNumberRequestedBeforeSca(MapRegexUtil.getDataRegex(context.getRequestData(), "TAN2Step6\\.OrderAccount\\.number"));
             setOrderReference(context);
-            paymentService.createPayment(context);
+            paymentService.createPayment(context, false);
+        }
+        return "response-templates/authenticated/custom-message-authorization-required-payment.json";
+    }
+
+    private String handleInstantPayment(HbciSandboxContext context) {
+        if (context.getBank().getSecurity().getPayment() == SensitiveAuthLevel.AUTHENTICATED) {
+            paymentService.createPaymentIfNeededAndPossibleFromContext(context);
+            return "response-templates/authenticated/custom-message-konto-mt940.json";
```

Comment:
Not sure if returning `custom-message-konto-mt940.json` is correct here, probably we want to return some payment-related file here - `custom-message-payment-response.json` is correct one (from authorized), same applies to handlePayment